contrib/qubes/doc/heads_udpate.md

```diff
@@ -0,0 +1,86 @@
+# Heads update
```

Comment:
file name has a typo in it (`udpate`)

Reply:
Fixed (changed here and in the file name)